io.gitlab.celleron56.libretile.yaml

```diff
@@ -0,0 +1,92 @@
+app-id: io.gitlab.celleron56.libretile
+runtime: org.freedesktop.Platform
+runtime-version: '22.08'
+sdk: org.freedesktop.Sdk
+command: /app/files/start.sh
```

Comment:
command should NOT be an absolute path. `start.sh` should be in the `PATH` instead.

Reply:
i have no idea where i should put the start.sh file so it is in PATH

Reply:
`/app/bin`, then it will work when set to `start.sh`.